### PR TITLE
Chore/update 202009

### PR DESCRIPTION
**Pull request by jrconlin**
## Description
Update dependencies to address cargo audit issues

## Testing

N/A

## Issue(s)

N/A

### CODE REVIEW

**Comment by jrconlin** [comment]
awesome: https://github.com/bodil/sized-chunks/issues/11

**Comment by pravic** [comment]
> # skipping audit pending https://github.com/bodil/sized-chunks/issues/11

There's `cargo audit --ignore ADVISORY_ID` that might help.

File: src/web/auth.rs [hunk 1]
```diff
@@ -206,8 +206,10 @@ pub fn hkdf_expand_32(info: &[u8], salt: Option<&[u8]>, key: &[u8]) -> ApiResult
 
 /// Helper function for [HMAC](https://tools.ietf.org/html/rfc2104) verification.
 fn verify_hmac(info: &[u8], key: &[u8], expected: &[u8]) -> ApiResult<()> {
-    let mut hmac: Hmac<Sha256> = Hmac::new_varkey(key)?;
-    hmac.input(info);
+    type HmacSha256 = Hmac<Sha256>;
+
+    let mut hmac: HmacSha256 = HmacSha256::new_varkey(key)?;
```
Comment (pjenvey) [inline src/web/auth.rs:211]: Let's kill the HmacSha256 type alias in these 3 places.

With us having to specify the type parameter at the `new_varkey` callsite it's superfluous to do it on the left side.

```suggestion
    let mut hmac = Hmac::<Sha256>::new_varkey(key)?;
```

**Review by pjenvey — CHANGES_REQUESTED** [review]

File: src/web/auth.rs (hunk 1 above)
Comment (pjenvey) [reply src/web/auth.rs:211]: ..and I just noticed the hmac docs' examples use the type alias. I really don't see a good reason why though 🤷‍♂️

Comment (jrconlin) [reply src/web/auth.rs:211]: Probably just legibility. I kinda liked the type for that reason, but yeah, better to be explicit.

Comment (pjenvey) [reply src/web/auth.rs]: Forgot the other 2 places

**Comment by tublitzed** [comment]
@jrconlin do we have a ticket associated with this PR?

**Review by pjenvey — APPROVED** [review]

**Pull request merged**